api/src/main/java/edu/cornell/mannlib/vitro/webapp/auth/requestedAction/AndAuthorizationRequest.java

```diff
@@ -0,0 +1,49 @@
+package edu.cornell.mannlib.vitro.webapp.auth.requestedAction;
+
+import java.util.Arrays;
+import java.util.List;
+
+import edu.cornell.mannlib.vitro.webapp.auth.attributes.AccessOperation;
+import edu.cornell.mannlib.vitro.webapp.auth.identifier.IdentifierBundle;
+import edu.cornell.mannlib.vitro.webapp.auth.objects.AccessObject;
+
+public class AndAuthorizationRequest extends AuthorizationRequest {
+
+    private final AuthorizationRequest ar1;
+    private final AuthorizationRequest ar2;
+
+    AndAuthorizationRequest(AuthorizationRequest ar1, AuthorizationRequest ar2) {
```

Comment:
why access modifier (public, protected) is not defined? You really want default access modifier here?

Reply:
I think I did. It is only should be used by AuthorizationRequest to be created by .and( ) method, so package visibility should be enough. Let me know if you think that should be changed.